nilearn/maskers/_utils.py

```diff
@@ -125,3 +128,34 @@ def concatenate_surface_images(imgs):
     output = SurfaceImage(mesh=imgs[0].mesh, data=output_data)
 
     return output
+
+
+def deconcatenate_surface_images(img):
```

Comment:
the surface utility functions from this module should probably be moved somewhere else (nilearn._utils.surface ?)